<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WelcomeWindow</class>
 <widget class="QMainWindow" name="WelcomeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WelcomeWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="singlePlayerBtn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>190</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Single Player</string>
    </property>
   </widget>
   <widget class="QPushButton" name="multiPlayerBtn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>310</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Multi Player</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitBtn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>430</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="easyBtn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Easy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backMenuBtn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>480</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Back to Menu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="normalBtn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>250</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Normal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hardBtn">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>370</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Hard</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
